packages/sanes-chrome-extension/src/routes/tx-request/index.tsx

```diff
@@ -17,9 +17,9 @@ const TxRequest = ({ location }: RouteComponentProps): JSX.Element => {
 
   const req = requestContext.firstRequest;
   checkRequest(req, location, toast);
-  const { data, accept, reject } = req!; // eslint-disable-line
+  const { senderUrl, responseData, accept, reject } = req!; // eslint-disable-line
```

Comment:
here too

Reply:
Refactored code to avoid the use of `!` entirely. Could you have another look?